src/utils/env.ts

```diff
@@ -29,20 +29,27 @@ export const notInProduction = <T = string>(key: T) => {
   }
 };
 
+// Compute these values first to get the right environment variables
+const frontEndpoint = loadEnv('ARENA_WEBSITE') || 'https://arena.utt.fr';
+const apiEndpointPort = loadIntEnv('API_PORT') || 3000;
+const apiEndpointPrefix = loadEnv('API_PREFIX') || '/';
+const apiEndpoint = `${frontEndpoint.replace(/:\d+$/, `:${apiEndpointPort}`)}${apiEndpointPrefix}`;
```

Comment:
Alors, on risque d'avoir une difficulté pour la production. L'api écoute sur le port 3000 dans un conteneur, mais la connection passe par un reverse proxy qui elle écoute sur le port 80/443.

Du coup, on ne peut pas générer l'api endpoint, il vaut mieux le laisser manuellement. ou bien faire:

```suggestion
const apiEndpoint = loadEnv('API_ENDPOINT') || `${frontEndpoint.replace(/:\d+$/, `:${apiEndpointPort}`)}${apiEndpointPrefix}`;
```

Reply:
Ouais mais il n'y a pas de port dans la prod dans le ARENA_WEBSITE donc le replace ne fait rien

Reply:
my bad, j'avais mal lu le `replace`